config/csv/config.go

```diff
@@ -33,6 +33,31 @@ func EmptyNull(emptyNull bool) ConfigFunc {
 	}
 }
 
+// MissingColumnNameAliasString set configs to rename empty column name with given string
```

Comment:
"MissingColumnNameAliasString..." -> "MissingColumnNameAlias sets the name to be used for empty columns..."
Perhaps add a comment here about the fact that you probably also want to set `AddColumnIndexToDuplicateColumns` if you expect to have multiple columns with missing names?